source/transactions/tests/retryable-abort-errorLabels.yml

```diff
@@ -0,0 +1,190 @@
+runOn:
+    -
+        minServerVersion: "4.3.0"
+        topology: ["replicaset, sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+tests:
+  - description: abortTransaction only retries once after RetryableWriteError from server
+    clientOptions:
+      retryWrites: false
```

Comment:
Is this relevant if the only insert we do is within a transaction? I realize it's in "abortTransaction only performs a single retry" but I don't understand why that test has it either. /cc @ShaneHarvey

Reply:
My thinking is that this was testing the line: `abortTransaction is a retryable write command. Drivers MUST retry once after abortTransaction fails with a retryable error according to the Retryable Writes Specification., regardless of whether retryWrites is set on the MongoClient or not.` from the transactions spec. I agree thought that it's not obvious how this is relevant to the test.

Reply:
/The test does not explicitly state that it is testing that abort still retries even if retryWrites is off.

Reply:
Yes the `abortTransaction only performs a single retry` test includes `retryWrites: false` to ensure that driver perform the retry regardless of the retryWrites setting. I don't think we need to set  `retryWrites: false` in these new tests as well.